aten/src/ATen/quantized/QTensorImpl.cpp

```diff
@@ -3,22 +3,20 @@
 namespace at {
 
 QTensorImpl::QTensorImpl(
-    Storage&& storage,
+    Storage storage,
```

Comment:
Why change this from an rvalue?

Reply:
Not sure it was a wrong optimisation or deliberately. It is better to revert the change.